coding-interviews/readme.md

```diff
@@ -0,0 +1,33 @@
+# Coding Interviews
+
+This section should be used as additional material after you have already gone through the following curriculum:
+- [algorithms](../algorithms/intro-to-algorithms.md)
+- [data-structures](../data-structures/intro-to-data-structures.md)
+
+Advisory Board Member Krys Flores wrote an entire page on her website about who to approach coding problems. We recommend reading it as a starting place. [Click here to read.](https://krysflores.com/coding-interviews)
+
+## Projected Time
+
+_Varies_
+
+## Data Structures and Algorithms 
+
+### Leetcode Style Problems
+
+Yes, sometimes you have to do leetcode problems. Almost no one thinks they're fun. Now that you've realized you have to look at Leetcode problems, you can get easily overwhelmed when you go to the Leetcode website. Don't randomly pick exercises. Instead, use [Grind 75](https://www.techinterviewhandbook.org/grind75) to curate a list for you based on the amount of time you want to spend studying.
+
+### Good Youtube Videos
+
+_Keep adding to this section! Different types of videos work for different types of people._
+
+- [Merge Intervals and Interval List Intersections by Krys Flores](https://youtu.be/baAxuwdISvc)
+- [Depth First Search in Trees by Krys Flores](https://www.youtube.com/watch?v=dG1s-HUdCPU)
+- [Dynamic Programing Patterns in Python by Krys Flores](https://www.youtube.com/watch?v=dG1s-HUdCPU)
+
+### Good books
+
+_Keep adding to this section! Different types of videos work for different types of people_
```

Comment:
@Yosolita1978 This would probably be a good place to add the books from the zoom chat. Do you have access to them? I can add here.

Reply:
I have them. I will drop here all the links that we can use as resources, feel free to re-organize them. 

[Grooking Algorithms](https://drive.google.com/file/d/1nzkdvaW1Gik8xHi74ViYFaT1PKhkXsME/view?usp=sharing)
[Algorithms (Sanjoy Dasgupta, Christos Padamitriou, Umesh Vazirani)](https://drive.google.com/file/d/1gyrhseQ7SVJSdUEGS2a0ndgaJrudAAgm/view?usp=sharing)
[A recorded session with Josh Goldberg about solving code challenges](https://www.dropbox.com/s/2yngfs5i9aj7m4c/video1849801289.mp4?dl=0)